src/js/nettest.js

```diff
@@ -66,10 +66,9 @@ NetworkTest.prototype = {
           newUrls.push(uri);
         } else if (
           uri.indexOf('?transport=') === -1 && uri.startsWith('turn')) {
-          newUrls.push(uri + '?' + transport);
+            newUrls.push(uri + '?' + transport);
         }
       }
-      iceServer.urls = newUrls;
```

Comment:
It only adding an empty object {'urls': []} which caused the tests to fail (RTCPeerconnection throws a malformed URL error). Not sure why it's needed.